dluhc-component-library/src/components/timetableForm/constants.ts

```diff
@@ -0,0 +1,23 @@
+import { NOT_STARTED } from "src/models/timetable/constants";
+import { FormState, Stage } from "./types";
+
+export const TITLE_KEY = "title";
+export const DESCRIPTION_KEY = "description";
+export const PUBLISH_DATE_KEY = "publishDate";
+export const SCOPING_KEY = "scoping";
+export const GATEWAY_1_KEY = "gateway1";
+
+const INITIAL_STAGE: Stage = {
+  startDate: { day: "", month: "", year: "" },
```

Comment:
`Non-blocker` This value is used in a few places could we pull it out into `DEFAULT_DATE` ?